packages/edit-site/src/components/editor/index.js

```diff
@@ -48,7 +47,8 @@ import useEditorTitle from './use-editor-title';
 import { useIsSiteEditorLoading } from '../layout/hooks';
 
 const { Editor, BackButton } = unlock( editorPrivateApis );
-const { useHistory, useLocation } = unlock( routerPrivateApis );
+const { useHistory, useLocation, isPreviewingTheme } =
```

Comment:
Router is a generic package that is meant to implement "routing" in any context. `isPreviewingTheme` is a site editor specific state, I'm not sure I understand this change.

Reply:
Based on what I know about https://github.com/WordPress/gutenberg/pull/63849#discussion_r1687870181 perhaps we should not consider `isPreviewingTheme` a site-editor concern, but also a front & post editor concern.

Reply:
I'll reply in the other comment then.